src/main/java/org/opensearch/security/auth/BackendRegistry.java

```diff
@@ -190,8 +191,11 @@ public boolean authenticate(final RestRequest request, final RestChannel channel
         final String sslPrincipal = (String) threadPool.getThreadContext().getTransient(ConfigConstants.OPENDISTRO_SECURITY_SSL_PRINCIPAL);
 
         if(adminDns.isAdminDN(sslPrincipal)) {
+            User adminUser = new User(sslPrincipal);
+            adminUser.setInternal(false);
```

Comment:
This line confuses me, but if we removed the 'isInternal' property I won't need to understand what this means

Reply:
I will remove this since it defaults to false

Reply:
Removed